sound/soc/intel/common/soc-acpi-intel-cnl-match.c

```diff
@@ -72,6 +72,12 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_cnl_sdw_machines[] = {
 		.sof_fw_filename = "sof-cnl.ri",
 		.sof_tplg_filename = "sof-cml-rt711-rt1308-rt715.tplg",
 	},
+	{
+		.link_mask = 0xB, /* 3 active links required */
+		.drv_name = "sdw_rt711_rt1308_rt715",
+		.sof_fw_filename = "sof-cnl.ri",
+		.sof_tplg_filename = "sof-cml-rt711-rt1308-rt715.tplg",
```

Comment:
I will merge this but we should probably use a different topology name here?

Reply:
Good idea.